java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/api/OpenmldbSession.scala

```diff
@@ -156,9 +156,7 @@ class OpenmldbSession {
     }
 
     var sql: String = sqlText
```

Comment:
Do not need to use `sql` now. Remove this and use `sqlText` directory.

Reply:
already edited